src/components/templates/PosterPage/components/ShareModal/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { FACEBOOK_APP_ID } from "secrets";
+import { PosterPageVenue } from "types/venues";
+import { WithId } from "utils/id";
+
+const getCategoriesFromVenue = (venue: WithId<PosterPageVenue>) => {
+  return venue?.poster?.categories || [];
+};
+
+export const getTitleTextForSharing = (
+  venue: WithId<PosterPageVenue>,
+  url: string
+) => {
```

Comment:
We should used named params here, not positional args.
```suggestion
export interface GetTitleTextForSharingProps {
  venue: WithId<PosterPageVenue>;
  url: string;
}
export const getTitleTextForSharing = ({ venue, url }: GetTitleTextForSharingProps) => {
```

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/9a27d5acb19bde1058ae95272a4ffae0a3b76283

Reply:
https://github.com/sparkletown/sparkle/pull/1396/commits/cb33a3f5ddfb6801e8af4f6bebc3d8bdedb72b69